missions/views/mission.py

```diff
@@ -27,14 +27,35 @@ class DetailView(LoginRequiredMixin, PermissionRequiredMixin, generic.DetailView
 
     def get_context_data(self, **kwargs):
         mission = self.get_object()
-        multiple_assigned_users = list()
-
-        for assignment in mission.assignment_set.all():
-            if assignment.user is not None:
-                intersection = [value for value in assignment.user.assignments.all() if value in mission.assignment_set.all()]
-                if len(intersection) > 1:
-                    multiple_assigned_users.append(assignment.user)
-        return {**super().get_context_data(**kwargs), 'multiple_assigned_users': multiple_assigned_users}
+        multiple_assigned_users = set()
+        users_overlaping_missions = set()
+
+        #for m in self.get_all_missions_with_overlaping_period(mission):
+        #    users_overlaping_missions.update(self.get_all_assigned_users(m))
+
+        multiple_assignments_in_mission = [a.user for a in mission.assignment_set.all()
```

Comment:
It is a lot faster to do this with queries on the database. As long as the dataset is **very** small it doesn't matter but mixing Python data structures with Queryset usually doesn't scale.